applications/ShapeOptimizationApplication/custom_utilities/response_functions/face_angle_response_function_utility.cpp

```diff
@@ -49,15 +49,13 @@ double FaceAngleResponseFunctionUtility::CalculateValue()
 {
 	KRATOS_TRY;
 
-	double value = 0.0;
-
-	for (auto& cond_i : mrModelPart.Conditions()){
-		const double g_i = CalculateConditionValue(cond_i);
+	const double value = block_for_each<SumReduction<double>>(mrModelPart.Conditions(), [&](Condition& rCond) {
```

Comment:
nice

but it seems you didn't include the parallel_utilties header

Reply:
its in the variable_utils.h already